Traceback (most recent call last):
  File "/home/mootjeloul/KuCoinCatcherBot/bot.py", line 1, in <module>
    from kucoin.client import Market
ModuleNotFoundError: No module named 'kucoin'
Traceback (most recent call last):
  File "/home/mootjeloul/KuCoinCatcherBot/bot.py", line 1, in <module>
    from kucoin.client import Market
ModuleNotFoundError: No module named 'kucoin'
Traceback (most recent call last):
  File "/home/mootjeloul/KuCoinCatcherBot/bot.py", line 1, in <module>
    from kucoin.client import Market
ModuleNotFoundError: No module named 'kucoin'
